spec/integration/alerts_spec.rb

```diff
@@ -286,6 +286,13 @@ module Hawkular::Alerts::RSpec
       expect(events).to_not be_nil
       expect(events.size).to be(0)
     end
+
+    it 'Should create an event' do
```

Comment:
The data in the cassette is stored in gzip form (https://github.com/pilhuhn/hawkular-client-ruby/blob/send_events/spec/vcr_cassettes/Alert/Events/Should_create_an_event.yml#L49) I guess, it's because the `vcr: { decode_compressed_response: true }` option is missing for the 'Alert/Events' (line 252)